src/python/pants/goal/run_tracker.py

```diff
@@ -353,6 +354,10 @@ def log(self, level, *msg_elements):
     """Log a message against the current workunit."""
     self.report.log(self._threadlocal.current_workunit, level, *msg_elements)
 
+  @classmethod
+  def _get_headers(cls):
+    return {'User-Agent': "Pants/v{}".format(version.VERSION)}
```

Comment:
Also, maybe make `Pants` lowercase. I'm not sure what the convention is but seems that requests uses lowercase.